### ISSUE TITLE
Configuration bootstrap error

### ISSUE THREAD

**Issue by cityji** [Snawoot/hola-proxy#123]
cant connect win 11 latest build using latest release :

logs : 

``` bash
MAIN    : 2024/10/06 12:00:13 main.go:196: WARNING  Detected latest extension version: "1.230.678". Pass -ext-ver parameter to skip resolve and speedup startup
MAIN    : 2024/10/06 12:00:13 main.go:204: INFO     hola-proxy client version v1.13.3 is starting...
MAIN    : 2024/10/06 12:00:13 main.go:205: INFO     Constructing fallback DNS upstream...
MAIN    : 2024/10/06 12:00:13 main.go:212: INFO     Initializing configuration provider...
CRED    : 2024/10/06 12:00:14 credservice.go:33: ERROR    Configuration bootstrap error: Post "https://client.hola.org/client_cgi/background_init?uuid=7d97cea9f32f4742a3137ff86f0e0ded": UClient handshake failed: tls: failed to verify certificate: x509: certificate signed by unknown authority. Retrying with the fallback mechanism...
CRED    : 2024/10/06 12:00:26 credservice.go:33: ERROR    Configuration bootstrap error: Post "https://client.hola.org/client_cgi/background_init?uuid=2ca8c0e94aa24a97839ee942e61687f5": context deadline exceeded. Retrying with the fallback mechanism...
CRED    : 2024/10/06 12:00:36 credservice.go:33: ERROR    Configuration bootstrap error: Post "https://client.hola.org/client_cgi/background_init?uuid=c995191960394453bf806ff845931853": context deadline exceeded. Retrying with the fallback mechanism...
CRED    : 2024/10/06 12:00:46 credservice.go:33: ERROR    Configuration bootstrap error: Post "https://client.hola.org/client_cgi/background_init?uuid=1ff21bacf5354733b30c433274e3c5a9": context deadline exceeded. Retrying with the fallback mechanism...
```

**Comment by Snawoot** [comment]
fix released in v1.13.4

**Comment by Snawoot** [comment]
There is no way to disable request on direct address, but you can lower the timeout from default 35s to something like 5s with option `-timeout 5s`.
